topologic/partitioning/louvain_stub.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import community
+
+
+# Documentation note: see docs/topologic.partitioning.rst for document string to louvain function alias
+louvain = community.best_partition
```

Comment:
There is one known problem with this route - help(topologic.partitioning) still shows the documentation from community.best_partition.  

Sphinx is the only thing able to take advantage of the function specification in the `rst` file.